sample-apps/react-native/dogfood/src/screens/Meeting/JoinMeetingScreen.tsx

```diff
@@ -24,107 +22,103 @@ type JoinMeetingScreenProps = NativeStackScreenProps<
   const [callId, setCallId] = useState<string>('');
-  const loopbackMyVideo = useAppGlobalStoreValue(
-    (store) => store.loopbackMyVideo,
-  );
+
   const { navigation } = props;
-  const appStoreSetState = useAppGlobalStoreSetState();
+  const userImageUrl = useAppGlobalStoreValue((store) => store.userImageUrl);
+  const username = useAppGlobalStoreValue((store) => store.username);
 
   const joinCallHandler = useCallback(() => {
     navigation.navigate('MeetingScreen', { callId });
   }, [navigation, callId]);
 
-  const handleCopyInviteLink = useCallback(
-    () =>
-      Clipboard.setString(
-        `https://stream-calls-dogfood.vercel.app/join/${callId}/`,
-      ),
-    [callId],
-  );
+  const startNewCallHandler = (call_id: string) => {
+    navigation.navigate('MeetingScreen', { callId: call_id });
+  };
 
   return (
-    <SafeAreaView style={styles.container}>
-      <View style={styles.headerContainer}>
-        <Text style={styles.headerText}>{'Whats the call ID?'}</Text>
-        <Button
-          title={'Randomise'}
-          color="blue"
-          onPress={() => {
-            const randomCallID = meetingId();
-            setCallId(randomCallID);
-          }}
-        />
+    <KeyboardAvoidingView
+      behavior={Platform.OS === 'ios' ? 'padding' : 'height'}
+      style={styles.container}
+    >
+      <Image source={{ uri: userImageUrl }} style={styles.logo} />
+      <View>
+        <Text style={styles.title}>Hello, {username}</Text>
+        <Text style={styles.subTitle}>
+          Start or join a meeting by entering the call ID.
+        </Text>
       </View>
-      <TextInput
-        style={styles.textInput}
-        placeholder={'Type your call ID here...'}
-        placeholderTextColor={'#8C8C8CFF'}
-        value={callId}
-        autoCapitalize="none"
-        autoCorrect={false}
-        onChangeText={(text) => {
-          setCallId(text.trim().split(' ').join('-'));
-        }}
-      />
-      <Button
-        title={'Create or Join call with callID: ' + callId}
-        color="blue"
-        disabled={!callId}
-        onPress={joinCallHandler}
-      />
-      <View style={styles.switchContainer}>
-        <Text style={styles.loopbackText}>Loopback my video(Debug Mode)</Text>
-        <Switch
-          value={loopbackMyVideo}
-          onChange={() => {
-            appStoreSetState((prevState) => ({
-              loopbackMyVideo: !prevState.loopbackMyVideo,
-            }));
+
+      <View style={styles.createCall}>
+        <TextInput
+          placeholder={'Type your Call ID'}
+          placeholderTextColor={'#8C8C8CFF'}
+          value={callId}
+          autoCapitalize="none"
+          autoCorrect={false}
+          onChangeText={(text) => {
+            setCallId(text.trim().split(' ').join('-'));
           }}
         />
+        <Button
+          onPress={joinCallHandler}
+          title="Join Call"
+          disabled={!callId}
```

Comment:
Doesn't work as far as I can see :/ call id is empty and still I'm being navigated to lobby with empty call id